### PR TITLE
InputManager: Fix for POINTERTAP firing when cursor is moved

### PR DESCRIPTION
**Pull request by PolygonalSun**
This PR contains a fix for the InputManager to not erroneously treat a `pointerup` event as a POINTERTAP if the cursor has moved outside of the drag threshold, even if it returns to the exact same point.

This fix should address this bug: https://github.com/BabylonJS/ThePirateCove/issues/168

### CODE REVIEW

**Review by sebavan — APPROVED** [review]

**Comment by sebavan** [comment]
Please wait for at least another reviewer before merging :-)

**Review by bghgary — APPROVED** [review]

**Pull request merged**
